Jenkinsfile

```
@@ -234,6 +234,49 @@ pipeline {
               set -e
               TEMPDIR=$(mktemp -d)
               docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH=master -v ${TEMPDIR}:/ansible/jenkins jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER} 
+              # Stage 1 - Jenkinsfile update
+              if [[ "$(md5sum Jenkinsfile | awk '{ print $1 }')" != "$(md5sum ${TEMPDIR}/docker-${CONTAINER_NAME}/Jenkinsfile | awk '{ print $1 }')" ]]; then
+                mkdir -p ${TEMPDIR}/repo
+                git clone https://github.com/${LS_USER}/${LS_REPO}.git ${TEMPDIR}/repo/${LS_REPO}
+                cd ${TEMPDIR}/repo/${LS_REPO}
+                git checkout -f master
```

Do we always want to use the master branch? Or should we
```suggestion
                git checkout -f "${BRANCH_NAME}"
```
Or would it be `git checkout -f {{ ls_branch }}` ?

---

it is, but this is the Jenkinsfile, not the Jenkinsfile.J2 template 😉